core-bundle/src/Resources/contao/templates/analytics/analytics_google.html5

```diff
@@ -1,16 +1,18 @@
 <?php
 
 // To use this script, please fill in your Google Analytics ID below
-$GoogleAnalyticsId = 'UA-XXXXX-X';
+// Example: 'G-XXXXXXX' (Google Analytics 4) or 'UA-XXXXX-X' (Universal Analytics)
+$GoogleAnalyticsId = 'XXXXX';
 
 // DO NOT EDIT ANYTHING BELOW THIS LINE UNLESS YOU KNOW WHAT YOU ARE DOING!
-if ('UA-XXXXX-X' != $GoogleAnalyticsId && !BE_USER_LOGGED_IN && !$this->hasAuthenticatedBackendUser()): ?>
+if ('XXXXX' != $GoogleAnalyticsId && !BE_USER_LOGGED_IN && !$this->hasAuthenticatedBackendUser()): ?>
 
+<script async src="https://www.googletagmanager.com/gtag/js?id=<?= $GoogleAnalyticsId ?>"></script>
 <script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-  ga('create', '<?= $GoogleAnalyticsId ?>', 'auto');
-  ga('set', 'anonymizeIp', true);
-  ga('send', 'pageview');
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
```

Comment:
```suggestion
  function gtag(){dataLayer.push(arguments)};
```

According to https://developers.google.com/tag-platform/tag-manager/web/datalayer#installation.

Reply:
They suggest my variant when you look for instructions directly while setting up a property 😁 But I suppose it doesn't matter.

![image](https://user-images.githubusercontent.com/10257912/184137106-a08e2188-e710-4da8-9ef6-c5963d9dea32.png)

Reply:
I see. 😄